src/framework/piral-core/src/types/utils.ts

```diff
@@ -1,6 +1,8 @@
-import { PiralEventMap } from 'piral-base';
```

Comment:
No we need that to ensure that `PiralEventMap` has been seen.

See below `declare module 'piral-base/lib/types' {`. This is crucial for declaration merging - otherwise, if the original was not yet seen (directly or indirectly) TS will only create a new interface - not merge it with the existing.

To ensure something has been seen we need the explicit import.

Reply:
adapted